bin/infection

```diff
@@ -6,9 +6,12 @@
  * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
  */
 $autoloaderInWorkingDirectory = getcwd() . '/vendor/autoload.php';
+$vendorDir = exec('composer config vendor-dir');
```

Comment:
So is it like “composer config...” will be executed every time someone wants to run Infection?

Reply:
Missed this, moved to elseif branch